### ISSUE TITLE
some of the new liquids don't behave like liquids (see desc)

### ISSUE THREAD

**Issue by ghost** [GrowthcraftCE/Growthcraft-1.7#129]
me and my friend we have Mystcraft on our server and we got lands with rivers and seas of Growthcraft liquids and I noticed some of them don't behave like liquids. You don't drown in them or if you fall you take fall damage.

For example 
Potent Fermented Sake 

I think you should create a test area for each liquid make sure it actually behaves like liquid when you dive into it.

**Comment by IceDragon200** [comment]
@trollworkout Blame minecraft, it specially handles water blocks which cause drowning, because of a bug regarding glass bottles and growthcraft liquids turning into water, I changed the material which breaks a lot of "water" like behaviour (drowning, taking fall damage)

I'll look into it, but no guarantee, if someone with more experience working with fluids appears that would be splendid!

**Comment by ghost** [comment]
@IceDragon200  just brining it to your attention. some liquids seem to work fine though. odd.

**Comment by IceDragon200** [comment]
@trollworkout I think I figured it out, I never set the Fluid properties, I'll try it later (after work about 17hr EST) and give you a build to mess with :hand:

**Comment by ghost** [comment]
@IceDragon200 sounds good :)

**Comment by IceDragon200** [comment]
@trollworkout Didn't even make a dent, fluids need to use Material.water if they want to be fluids...

I've added a config option to change booze fluids to "water" material, this re-introduces the water bottle problem, but it behaves like a proper fluid.

You just can't win :(

A. Fluids work like fluids (you can swim in them and drown, but bottles will treat every fluid as water)
B. Fluids are booze (you can move in them, but you can't swim, bottles don't do jack s**t to them)

So new config option in `growthcraft/cellar.conf`

```
"booze fluid" {
    B:"Booze blocks use water material"=true
}
```

tl;dr 
68c96a1 [Build](https://dl.dropboxusercontent.com/u/72618186/minecraft/mods/growthcraft/2.3.1/growthcraft-1.7.10-2.3.1-complete--68c96a1.jar)

**Comment by IceDragon200** [comment]
@trollworkout Seeing as there is no one fix to solve everything, can we close this now?

**Comment by ghost** [comment]
@IceDragon200  as long as you can swim and drown in liquids is all good. just disable right click with water bottle and use bucket crafting instead.

**Comment by IceDragon200** [comment]
@trollworkout I'm sorry dave, I'm afraid can't do that.
- In order for liquids to behave like "liquids", they have to use the `Material.water`.
- Using `Material.water` causes the bottle bug (which is a Minecraft implementation issue, I CANT FIX THAT)
- Using any other material for the fluids, causes them to behave differently (non-liquid) (Swimming and so forth, are hardcoded for Material.water, once again, this is a Minecraft thing, and I CANT FIX IT)

tl;dr Minecraft has buggy core-code.

**Comment by ghost** [comment]
@IceDragon200 Just close it for now but it may come back at a later time. I'll also post this water bottle issue on forge github since they kinda need to implement it.
